core/lib/src/response/response.rs

```diff
@@ -1018,7 +1080,8 @@ impl<'r> Response<'r> {
 
     /// Sets the body of `self` to be the fixed-sized `body`. The size of the
     /// body is obtained by `seek`ing to the end and then `seek`ing back to the
-    /// start.
+    /// start. Since this is an asynchronous operation, it returns a promise
```

Comment:
promise -> `Future`. We can probably leave out the part about "holds onto this Response" as well, since that's the case for all `self`-taking `async fn`.